internal/v1/handler_compose_image.go

```diff
@@ -24,6 +26,29 @@ import (
 	"github.com/labstack/echo/v4"
 )
 
+const (
+	AAP_FIRST_BOOT_FILENAME = "aap-first-boot-reg"
+	AAP_FIRST_BOOT_SERVICE  = `
+Description=Ansible Automations Platform first boot registration
+ConditionFileIsExecutable=/usr/local/sbin/aap-first-boot-reg
+ConditionPathExists=!/var/local/.aap-first-boot-reg-done
+Wants=network-online.target
+After=network-online.target
+After=osbuild-first-boot.service
+
+[Service]
+Type=oneshot
+ExecStart=/usr/local/sbin/aap-first-boot-reg
+ExecStartPost=/usr/bin/touch /var/local/.aap-first-boot-reg-done
+RemainAfterExit=yes
+
+[Install]
+WantedBy=basic.target
+WantedBy=multi-user.target
+WantedBy=graphical.target
+`
+)
```

Comment:
Can you drop both basic and graphical? This is a leftover I am going to fix in the UI.

I also suggest to make use of `embed` package and extract this into `aap-first-boot.tmpl.service` so we will get syntax hightlighting benefits. It could have been generalized via attributes in the future as well into a generic template for all firstboot templates.